ocs_ci/ocs/ui/views.py

```diff
@@ -384,6 +384,63 @@
     "choose_kubeconfig": ("//button[text()='Kubeconfig']", By.XPATH),
     "Kubeconfig_text": ('textarea[label="Kubeconfig"]', By.CSS_SELECTOR),
     "Submit_import": ("//button[text()='Import']", By.XPATH),
+    "cluster-sets": ("//a[normalize-space()='Cluster sets']", By.XPATH),
```

Comment:
Can we split the `acm_page_nav dic` to  `acm_page_nav` and `acm_configuration`?

Reply:
done